client/app/bundles/course/assessment/pages/AssessmentsIndex/AssessmentsTable.tsx

```diff
@@ -39,7 +41,19 @@ const AssessmentsTable = (props: AssessmentsTableProps): JSX.Element => {
             </Link>
           </label>
 
-          {!display.isStudent && <StatusBadges for={assessment} />}
+          <div className="flex items-center">
+            {assessment.timeLimit && (
```

Comment:
Why not all these into statusbadges?

Reply:
I wanted to show this tooltip for being timed to teacher as well.

Reply:
Then move the isStudent logic inside the status badges?

Reply:
done. also refactored the status badges to cater with this new logic